pom.xml

```diff
@@ -19,6 +19,7 @@
 		<hamcrest.version>2.2</hamcrest.version>
 		<maven-jar-plugin.version>3.1.2</maven-jar-plugin.version>
 		<maven-surefire-plugin.version>3.0.0-M5</maven-surefire-plugin.version>
+                <maven-failsafe-plugin.version>3.0.0-M5</maven-failsafe-plugin.version>
```

Comment:
Should this have the same alignment as the other lines? Or is it misaligned for some reason only in the diff?

Reply:
yeah, something went wrong there, fixed now, thanks!